### ISSUE TITLE
Add the Social Login with JWTAuth

### ISSUE THREAD

**Issue by rluders** [rluders/wn-jwtauth-plugin#28]
Adds the support to login using OAuth providers, like Facebook, Google, Instagram, and Twitter, but still using the JWT Token.

**Comment by thomas4Bitcraft** [comment]
Any update on this?

**Comment by rluders** [comment]
@thomas4Bitcraft not really. I didn't have time to work in this plugin anymore. But if you want to implement this feature, I'll be happy to review and accept your PR.
